README.md

```diff
@@ -1,37 +1,92 @@
-# VirtoCommerce.Pricing
-VirtoCommerce.Pricing module represents pricing management system.
-Key features:
-* multiple price lists
-* dynamic price evaluation
-* price list assignment based on dynamic conditions
+---
+author: Maksim Kopnov
+date: 4/15/2019
+---
+# Pricing Module
 
-![Pricing UI](https://cloud.githubusercontent.com/assets/5801549/15568725/ade88e2a-232d-11e6-9486-9bbae8f15fa8.png)
+## Overview
 
+The **Pricing Module** is designed for storing, managing and evaluating product prices.
 
-# Documentation
-User guide: <a href="https://virtocommerce.com/docs/vc2userguide/price-lists" target="_blank">Price Lists</a>
+The Pricing Module is consisted of three main objects:
 
-Developer guide:
+- Price
+- Pricelist
+- Pricelist Assignment
 
-# Installation
-Installing the module:
-* Automatically: in VC Manager go to Configuration -> Modules -> Pricing module -> Install
-* Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-pricing/releases. In VC Manager go to Configuration -> Modules -> Advanced -> upload module package -> Install.
+### Price list
```

Comment:
@OlegoO Do we have someone with fluent English to check grammar, articles, etc. and overall text style?

Reply:
@yecli  should be Price
@mvktsk  we will check it later

Reply:
To avoid misunderstanding, if I haven't expressed it precisely - it is not about proper English, it is about whole header sense. So the question is not "Price list" or "Prices list", but "Price" or "Prices" vs "Price list", as it should be based on header logic - every header corresponds to one element in the list above:
- Price
- Pricelist
- Pricelist Assignment

Reply:
I've updated the document

Reply:
@OlegoO I've update pull request